v2/src/App.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import './App.css';
+import { ActiveTransit, TransitIconImages } from './Interfaces';
+import logo from './assets/trajectory_logo.svg';
+import buttonDown from './assets/button-down.svg';
+import GOTrainLight from './assets/gotrain-light.svg';
+import VIALight from './assets/via-light.svg';
+import StreetCarLight from './assets/streetcar-light.svg';
+import SubwayLight from './assets/subway-light.svg';
+import BusLight from './assets/bus-light.svg';
+import searchLight from './assets/search-light.svg';
+
+const samepleVehicles = [
```

Comment:
Is there an interface for this?

Reply:
This is just sample data to play with the carousels, don't think we need one

Reply:
Once we get actual data we'll be able to reuse the interface. Would be useful to define it right away I think

Reply:
Done 👍